Tensile/Source/client/source/Reference.cpp

```diff
@@ -175,9 +177,12 @@ namespace Tensile
                 }
             }
 
-            // Enable dynamic thread adjustment and set num threads to avoid defaulting to 1
+            // Set num threads and enable proc_bind to prevent main thread from migrating
             // Defaults cause validation to affect benchmark timing
-            omp_set_dynamic(1);
+            // Dynamic thread mode causes HostLibraryTests to fail
+            const char* env_bind = std::getenv("OMP_PROC_BIND");
```

Comment:
Do we ignore the case OMP_PROC_BIND is already defined and the value is not true?

Reply:
If it's already defined, then we keep the original value. This only sets it if user didn't request a specific OMP_PROC_BIND.

Reply:
I am not sure if we need to care such case or not, but the issue might not be solved if OMP_PROC_BIND is defined as false.
Is that correct?

Reply:
That's correct. I figured if someone manually set OMP_PROC_BIND to false then that is the behaviour they want. More likely they might set OMP_PROC_BIND=close or spread to keep threads from migrating, but affect distribution.

Reply:
Thanks for confirmation. That should be OK.